native/src/snappyclient/cpp/impl/ClientService.cpp

```diff
@@ -1938,6 +1938,13 @@ boost::shared_ptr<TSSLSocket> ClientService::createSSLSocket(
   if (!propVal.empty()) {
     sslSocketFactory.loadTrustedCertificates(propVal.c_str());
   }
+  else {
+    sslProperty = this->getSSLPropertyName(SSLProperty::USESYSTEMSTORE);
+    propVal = this->getSSLPropertyValue(sslProperty);
+    if (!propVal.empty() && boost::iequals(propVal, "true")) {
```

Comment:
shouldn't this be true by default i.e. if (propVal.empty() || boost::iequals(propVal, "true"))